components/mpas-framework/src/tools/registry/gen_inc.c

```diff
@@ -1618,6 +1731,86 @@ int parse_var(FILE *fd, ezxml_t registry, ezxml_t superStruct, ezxml_t currentVa
 			fortprintf(fd, "      call mpas_add_att(%s %% attLists(1) %% attList, 'units', '%s')\n", pointer_name_arr, temp_str);
 		}
 
+		if ( varbounds != NULL ) {
+			string = strdup(varbounds);
+			tofree = string;
+
+			token = strsep(&string, "'");
+			sprintf(temp_str, "%s", token);
+
+			while ( ( token = strsep(&string, "'") ) != NULL ) {
+				sprintf(temp_str, "%s''%s", temp_str, token);
+			}
+
+			free(tofree);
+
+			fortprintf(fd, "         call mpas_add_att(%s %% attLists(const_index) %% attList, 'bounds', '%s')\n", pointer_name_arr, temp_str);
```

Comment:
Here and for the remaining attributes except `standard_name`, I think this is over-indented compared with adjacent code.  Here's what I'm seeing:
```
      call mpas_add_att(r1aPtr(1) % attLists(1) % attList, 'units', 'm')
         call mpas_add_att(r1aPtr(1) % attLists(const_index) % attList, 'bounds', 'lonVertex')
         call mpas_add_att(r1aPtr(1) % attLists(const_index) % attList, 'cell_measures', 'area: areaCell')
         call mpas_add_att(r1aPtr(1) % attLists(const_index) % attList, 'cell_method', 'area: mean where sea')
         call mpas_add_att(r1aPtr(1) % attLists(const_index) % attList, 'coordinates', 'lonCell latCell')
      call mpas_add_att(r1aPtr(1) % attLists(1) % attList, 'standard_name', 'sea_surface_height_above_geoid')

```

Reply:
Also, I think `const_index` needs to be `1` for a `var`.  See above and below.